setup.py

```diff
@@ -28,19 +28,22 @@
 ]
 launch_requirements = [
     "awscli",
-    "nbconvert",
-    "nbformat",
-    "chardet",
-    "iso8601",
-    "typing_extensions",
+    "azure-identity",
+    "azure-containerregistry",
+    "azure-storage-blob",
     "boto3",
     "botocore",
+    "chardet",
     "google-auth",
+    "google-cloud-artifact-registry",
     "google-cloud-compute",
     "google-cloud-storage",
-    "google-cloud-artifact-registry",
+    "iso8601",
     "kubernetes",
     "optuna",
+    "nbconvert",
+    "nbformat",
+    "typing_extensions",
```

Comment:
👍 thanks for sorting these

Reply:
launch requirements are getting big, we may want to consider splitting these into env specific cases...

Reply:
https://wandb.atlassian.net/browse/WB-14051